### PR TITLE
Renamed prefix SV to SU (2020R1)

### LINKED ISSUE #17: Rename prefix SV to SU (2020R1)
Update all pages and references to use SU instead of SV in 2020R1 branch.

### PR DESCRIPTION
**Pull request by markoan**
Renamed prefix SV to SU in latest commit for branch 2019R2
- Updated zip
- Renamed all pages and set references to correct name and folder
- Updated GI
- Updated Dashboard
- Updated mobile definition

Closes #17

### CODE REVIEW

File: AcumaticaSurveyPkg/Customization/Pages/SU/SU201000.aspx.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@
 using System.Web.UI.WebControls;
 using System.Web.UI.HtmlControls;
 
-public partial class  Page_SV301000 : PX.Web.UI.PXPage
+public partial class  Page_SU201000 : PX.Web.UI.PXPage
```
Comment (DhirenChhapgar) [inline AcumaticaSurveyPkg/Customization/Pages/SU/SU201000.aspx.cs:10]: @markoan this should be SU301000

File: AcumaticaSurveyPkg/Customization/Pages/SU/SU301000.aspx.cs [hunk 2]
```diff
@@ -7,7 +7,7 @@
 using System.Web.UI.WebControls;
 using System.Web.UI.HtmlControls;
 
-public partial class  Page_SV201000 : PX.Web.UI.PXPage
+public partial class  Page_SU301000 : PX.Web.UI.PXPage
```
Comment (DhirenChhapgar) [inline AcumaticaSurveyPkg/Customization/Pages/SU/SU301000.aspx.cs:10]: @markoan this should be SU201000

File: AcumaticaSurveyPkg/Customization/_project/ProjectMetadata.xml [hunk 3]
```diff
@@ -1,2 +1,2 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<project name="AcumaticaSurveyPkg" level="0" description="" />
\ No newline at end of file
+<project name="Customization" level="0" description="" />
```
Comment (DhirenChhapgar) [inline AcumaticaSurveyPkg/Customization/_project/ProjectMetadata.xml:2]: @markoan we need to keep AcumaticaSurveyPkg as project name

Comment (markoan) [reply AcumaticaSurveyPkg/Customization/_project/ProjectMetadata.xml]: Done in latest commit.

**Pull request merged**